edx_rest_framework_extensions/auth/jwt/tests/test_authentication.py

```diff
@@ -751,6 +757,121 @@ def test_authenticate_jwt_and_no_session_and_set_request_user(self, mock_set_cus
         assert 'jwt_auth_get_lms_user_id_status' not in set_custom_attribute_keys
         assert response.status_code == 200
 
+    @override_settings(
+        EDX_DRF_EXTENSIONS={
+            ENABLE_JWT_AND_LMS_USER_EMAIL_MATCH: False
+        },
```

Comment:
[nit] It might make sense to drop this so we can also test that False is the default.